utils/index.d.ts

```diff
@@ -880,6 +880,55 @@ export interface IWizardOptions<T extends IActionContext> {
     showLoadingPrompt?: boolean;
 }
 
+export interface ActivityTreeItemOptions {
+    label: string;
+    contextValuePostfix?: string;
+    collapsibleState?: TreeItemCollapsibleState;
+    children?: (parent: AzExtParentTreeItem) => AzExtTreeItem[];
```

Comment:
Maybe `getChildren` since it's a method?
```suggestion
    getChildren?: (parent: AzExtParentTreeItem) => AzExtTreeItem[];
```